Resources/Locale/en-US/armor/armor-examine.ftl

```diff
@@ -4,3 +4,9 @@ armor-examinable-verb-message = Examine the armor values.
 armor-examine = It provides the following protection:
 armor-coefficient-value = - [color=yellow]{$type}[/color] damage reduced by [color=lightblue]{$value}%[/color].
 armor-reduction-value = - [color=yellow]{$type}[/color] damage reduced by [color=lightblue]{$value}[/color].
+armor-damage-type-blunt = Blunt
+armor-damage-type-slash = Slash
+armor-damage-type-piercing = Piercing
+armor-damage-type-heat = Heat
+armor-damage-type-radiation = Radiation
+armor-damage-type-caustic = Caustic
```

Comment:
you're missing strings for bloodloss, asphyxiation, cellular, cold, poison, shock, and structural.